app/CocoaPods.xcodeproj/project.pbxproj

```diff
@@ -614,11 +614,11 @@
 				TargetAttributes = {
 					510254221C0A60670045FCD8 = {
 						CreatedOnToolsVersion = 7.1;
-						DevelopmentTeam = 4R4XY9R8JR;
+						DevelopmentTeam = GN257VN844;
```

Comment:
yeah, that's what I was expecting, still trying to get sparkle working with the app transport stuff ATM though